certora/conf/v1.5/guards.conf

```diff
@@ -0,0 +1,37 @@
+// a conf file for safe module guards
+{
+    "files": [
+        "certora/harnesses/SafeHarness.sol",
+        "certora/mocks/ModuleGuardMock.sol", // a module guard 
+        "certora/mocks/TxnGuardMock.sol", // a (safe) guard
+    ],
+    "link": [
+
+    ],
+    "verify": "SafeHarness:certora/specs/v1.5/Guards.spec", 
+
+    "assert_autofinder_success": true,
+    "optimistic_summary_recursion": true,
+    "optimistic_contract_recursion": true,
+    
+    "summary_recursion_limit": "2",
+    "contract_recursion_limit": "2",
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "optimistic_hashing": true,
+    "optimistic_fallback": true,
+    
+    "rule_sanity": "basic",
+    
+    "solc": "solc7.6",
+    "solc_via_ir": false,  
+
+    "java_args": [
+        " -ea -Dlevel.setup.helpers=info"
+    ],
+    "process": "emv",
+    "prover_args": [
+        " -verifyCache -verifyTACDumps -testMode -checkRuleDigest -callTraceHardFail on",
```

Comment:
ditto

Reply:
getting rid of prover args actually makes it so two rules are violated: see links to [the run with prover args](https://prover.certora.com/output/39601/d0bd5f2c7f9b4cd99d02ba1151414e2a?anonymousKey=c62c582b54df0f4c7ed790fcdc10c89605289edc) and [the run without prover args](https://prover.certora.com/output/39601/d9fd32e7179b4516ace9dd44f4da0168?anonymousKey=69d631075a927e6147867bdae252b745ecc2015f).

I just got these from the setup Dominik gave me. Does that raise any red flags for you?